mappings/net/minecraft/block/StoneSlab2BaseBlock.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_5216 net/minecraft/block/StoneSlab2BaseBlock
```

Comment:
what

Reply:
Base class for `StoneSlab2Block` and `DoubleStoneSlab2Block`

Reply:
ah ok

Reply:
AbstractStoneSlab2Block ?